discovery-provider/src/queries/get_feed_es.py

```diff
@@ -266,7 +266,7 @@ def fetch_followed_saves_and_reposts(current_user_id, item_keys, limit):
                 ]
             }
         },
-        "size": limit * 20,  # how much to overfetch?
+        "size": limit * 10,  # how much to overfetch?
```

Comment:
is 10 sufficient? / maybe a comment explaining the napkin support for 10

Reply:
Removed the over fetch for:
https://www.elastic.co/guide/en/elasticsearch/reference/current/collapse-search-results.html